CMakeLists.txt

```diff
@@ -67,6 +67,8 @@ option(NRN_DYNAMIC_UNITS_USE_LEGACY "Use legacy units as default for dynamic uni
 option(NRN_ENABLE_MOD_COMPATIBILITY "Enable CoreNEURON compatibility for MOD files" ${NRN_ENABLE_MOD_COMPATIBILITY_DEFAULT})
 option(NRN_ENABLE_REL_RPATH "Use relative RPATH in binaries. for relocatable installs/Python" ${NRN_ENABLE_REL_RPATH_DEFAULT})
 option(NRN_WHEEL_STATIC_READLINE "Use static readline libraries for the wheels." ${NRN_WHEEL_STATIC_READLINE_DEFAULT})
+option(NRN_ENABLE_CALIPER_PROFILING "Enable caliper profiling provided by CoreNEURON" OFF)
+option(NRN_ENABLE_LIKWID_PROFILING "Enable likwid profiling  provided by CoreNEURON" OFF)
```

Comment:
Just a quick comment: while moving forward with the profiling options in CMake, I think we should change this to a single option with different values i.e. `-DNRN_ENABLE_PROFILING=LIKWID|CALIPER|VTUNE` and so on.